docs/source/optimisation.rst

```diff
@@ -119,8 +119,27 @@ LADMM
    :members:
    :inherited-members: run, update_objective_interval, max_iteration
 
+
+
+Algorithms (Stochastic)
+========================
+
+There is a growing range of Stochastic optimisation algorithms available with potential benefits of faster convergence in number of iterations or in computational cost. 
+This is an area of continued development for CIL.  
+
+
+
 SPDHG
 -----
+Stochastic Primal Dual Hybrid Gradient (SPDHG) is a stochastic version of PDHG and deals with optimisation problems of the form: 
+    
+    .. math::
+    
+      \min_{x} f(Kx) + g(x) = \min_{x} \sum f_i(K_i x) + g(x)
+
+by passing a sampler (e.g. of the CIL Sampler class) each iteration considers just one index of the sum reducing computational cost. For more examples see our [user notebooks]( https://github.com/vais-ral/CIL-Demos/blob/master/Tomography/Simulated/Single%20Channel/PDHG_vs_SPDHG.py).
```

Comment:
I don't think this is implemented: SPDHG does not accept a sampler and will do only stochastic with replacement